cmd/commands/runtime.go

```diff
@@ -2699,30 +2728,15 @@ func postInstallationHandler(ctx context.Context, opts *RuntimeInstallOptions, e
 	if err != nil && !*disableRollback {
 		summaryArr = append(summaryArr, summaryLog{"----------Uninstalling runtime----------", Info})
 		log.G(ctx).Warnf("installation failed due to error : %s, performing installation rollback", err.Error())
-		iscCloneOpts := &git.CloneOptions{
-			FS:               fs.Create(memfs.New()),
-			CreateIfNotExist: false,
-		}
-		iscCloneOpts.Repo, err = getIscRepo(ctx)
-		handleCliStep(reporter.UninstallStepGetIscRepoFromPlatform, "Getting isc repo from platform before rollback", err, false, true)
-		if err != nil {
-			log.G(ctx).Errorf("failed to get isc repo from platform before installation rollback: %s", err.Error())
-		}
-
-		inferProviderFromRepo(opts.InsCloneOpts)
-		iscCloneOpts.Auth = opts.InsCloneOpts.Auth
-		iscCloneOpts.Progress = opts.InsCloneOpts.Progress
-		iscCloneOpts.Parse()
```

Comment:
Why do we not need this anymore?

Reply:
i've moved the isc removal to app-proxy so no need for isc clone opts